RecoTauTag/RecoTau/plugins/PFRecoTauChargedHadronProducer.cc

```diff
@@ -164,6 +168,9 @@ void PFRecoTauChargedHadronProducer::produce(edm::Event& evt, const edm::EventSe
   // loop over our jets
   BOOST_FOREACH( const reco::PFJetRef& pfJet, pfJets ) {
     
+    if(pfJet->pt() - minJetPt_ < 1e-5) continue;
+    if(std::abs(pfJet->eta()) - maxJetAbsEta_ > -1e-5) continue;
```

Comment:
Hi,
I agree with the comment, but this is to be consistent with already existing code here:
https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/RecoTauTag/RecoTau/plugins/RecoTauProducer.cc#L168 and L169. All this can be corrected to the "standard expression" of pt<minPt.